AcsEmulator/AcsEmulatorCLI/AcsEmulatorCLI.csproj

```diff
@@ -26,7 +26,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <None Include="..\AcsEmulatorAPI\trouter_selfSigned.pfx" CopyToOutputDirectory="Always"></None>
+    <None Condition="Exists('..\AcsEmulatorAPI\trouter_selfSigned.pfx')" Include="..\AcsEmulatorAPI\trouter_selfSigned.pfx" CopyToOutputDirectory="Always"></None>
```

Comment:
Oh, I renamed that file, could you update it according to the  ps1 script?

Reply:
Sure! I was actually not sure about that (as ps1 to create that other did not exists), let me rename quickly, but it will require also some change to the CLI project that still reference that trouter pfx in console messages